client/components/Contact.scss

```diff
@@ -103,34 +97,84 @@
         }
 
         &__field {
-          font-family: Roboto;
           font-size: 18px;
           width: 100%;
           border-radius: 4px;
           line-height: 1.56;
-          margin: 0px 0px 20px 0px;
+          margin: 0px 0px 8px 0px;
           border: solid 1px #c9c9c9;
-          background-color: var(--white);
+          background-color: #ffffff;
```

Comment:
White is the default background color, so does not need to be set explicitly here.